python-sdk/src/astro/sql/operators/base_decorator.py

```diff
@@ -188,7 +188,7 @@ def translate_jinja_to_sqlalchemy_template(self, context: dict) -> None:
                 context[k] = jinja_table_identifier
                 self.parameters[k] = jinja_table_parameter_value
             else:
-                context[k] = ":" + k
+                context[k] = self.database_impl.parameterize_variable(k)
```

Comment:
ok cool, I now know what that database function was about -- but can you still add description about that in the BaseDatabase class please